lib/algolia/client.rb

```diff
@@ -697,32 +719,34 @@ def Algolia.get_user_key(key, request_options = {})
   #    - settings : allows to get index settings (https only)
   #    - editSettings : allows to change index settings (https only)
   #
-  #  @param obj can be two different parameters:
+  #  @param object can be two different parameters:
   #        The list of parameters for this key. Defined by a NSDictionary that
   #        can contains the following values:
   #          - acl: array of string
   #          - indexes: array of string
   #          - validity: int
   #          - referers: array of string
   #          - description: string
-  #          - maxHitsPerQuery: integer
+  #          - max_hits_per_query: integer
   #          - queryParameters: string
-  #          - maxQueriesPerIPPerHour: integer
+  #          - max_queries_per_IP_per_hour: integer
   #  @param request_options contains extra parameters to send with your query - Default = {}
   #
-  def Algolia.add_api_key(obj, request_options = {}, maxQueriesPerIPPerHour = 0, maxHitsPerQuery = 0, indexes = nil)
-    Algolia.client.add_api_key(obj, request_options, maxQueriesPerIPPerHour, maxHitsPerQuery, indexes)
+  def Algolia.add_api_key(object, request_options = {}, max_queries_per_IP_per_hour = 0, max_hits_per_query = 0, indexes = nil)
```

Comment:
Isn't renaming optional parameter a break of backward compat'? I might be mixing up with Python.

Reply:
Nope, you can even do `add_api_key(plop = 'hello')` it will work; the order is important though